eaf.el

```diff
@@ -549,7 +547,7 @@ Please ONLY use `eaf-bind-key' to edit EAF keybindings!"
         ;; Send key to Python side if key-command is single character key.
         ((or (equal key-command "self-insert-command")
              (equal key-command "completion-select-if-within-overlay")
-             (equal key-command "nil")
+             (eq key-command 'nil)
```

Comment:
symbol-name always return "nil" string, 
Have you test this change?

Reply:
I tested it and don't see any problem. `eq` compares the object `nil` and `key-command` right? If you feel unsafe, I can change it back.

Reply:
I'm sure symbol-name always return string (eq "nil" 'nil) is nil , it's wrong